native FUSE FS broken

**Describe the bug**
Fails to build when using FUSE_FS_ACCESS=y in native_sim
```
-- west build: building application
[1/145] Preparing syscall dependency handling

[2/145] Generating include/generated/zephyr/version.h
-- Zephyr version: 4.1.99 (/home/user/zephyrproject/zephyr), build: v4.1.0-267-gac24d2277e02
[143/145] Linking C executable zephyr/zephyr.elf
Generating files from /home/user/zephyrproject/zephyr/build/zephyr/zephyr.elf for board: native_sim
[144/145] Building native simulator runner, and linking final executable
FAILED: zephyr/CMakeFiles/native_runner_executable zephyr/zephyr.exe /home/user/zephyrproject/zephyr/build/zephyr/CMakeFiles/native_runner_executable /home/user/zephyrproject/zephyr/build/zephyr/zephyr.exe 
cd /home/user/zephyrproject/zephyr/build/zephyr && /usr/bin/make -f /home/user/zephyrproject/zephyr/scripts/native_simulator/Makefile all --warn-undefined-variables -r NSI_CONFIG_FILE=/home/user/zephyrproject/zephyr/build/zephyr/NSI/nsi_config
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘ffa_run_pending_op’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:65:30: warning: implicit declaration of function ‘offsetof’ [-Wimplicit-function-declaration]
   65 | #define OP_RMDIR             offsetof(struct ffa_op_callbacks, rmdir)
      |                              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:113:14: note: in expansion of macro ‘OP_RMDIR’
  113 |         case OP_RMDIR:
      |              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:33:1: note: ‘offsetof’ is defined in header ‘<stddef.h>’; did you forget to ‘#include <stddef.h>’?
   32 | #include "fuse_fs_access_bottom.h"
  +++ |+#include <stddef.h>
   33 | 
```

- What target platform are you using? native_sim
- What have you tried to diagnose or workaround this issue? git bisect
- Is this a regression? Yes ac24d2277e021a92d435efc45f9121e00970d0cd is the first bad commit

**To Reproduce**
Steps to reproduce the behavior:
```
west init ~/zephyrproject
cd zephyrproject/
west update
cd ~/zephyrproject/zephyr
west sdk install
west build -p auto -b native_sim samples/subsys/shell/fs -- -DCONFIG_FUSE_FS_ACCESS=y
```

**Expected behavior**
Successful build

**Impact**
native_sim cannot build when FUSE_FS_ACCESS=y

**Logs and console output**
<details>

```
-- west build: making build dir /home/user/zephyrproject/zephyr/build pristine
-- west build: generating a build system
Loading Zephyr default modules (Zephyr base).
-- Application: /home/user/zephyrproject/zephyr/samples/subsys/shell/fs
-- CMake version: 3.22.1
-- Found Python3: /usr/bin/python3 (found suitable version "3.10.12", minimum required is "3.10") found components: Interpreter 
-- Cache files will be written to: /home/user/.cache/zephyr
-- Zephyr version: 4.1.99 (/home/user/zephyrproject/zephyr)
-- Found west (found suitable version "1.3.0", minimum required is "0.14.0")
-- Board: native_sim, qualifiers: native
-- ZEPHYR_TOOLCHAIN_VARIANT not set, trying to locate Zephyr SDK
-- Found host-tools: zephyr 0.17.1 (/home/user/zephyr-sdk-0.17.1)
-- Found toolchain: host (gcc/ld)
-- Found Dtc: /home/user/zephyr-sdk-0.17.1/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.7.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/user/zephyrproject/zephyr/boards/native/native_sim/native_sim.dts
-- Generated zephyr.dts: /home/user/zephyrproject/zephyr/build/zephyr/zephyr.dts
-- Generated pickled edt: /home/user/zephyrproject/zephyr/build/zephyr/edt.pickle
-- Generated devicetree_generated.h: /home/user/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/devicetree_generated.h
Parsing /home/user/zephyrproject/zephyr/Kconfig
Loaded configuration '/home/user/zephyrproject/zephyr/boards/native/native_sim/native_sim_defconfig'
Merged configuration '/home/user/zephyrproject/zephyr/samples/subsys/shell/fs/prj.conf'
Merged configuration '/home/user/zephyrproject/zephyr/build/zephyr/misc/generated/extra_kconfig_options.conf'
Configuration saved to '/home/user/zephyrproject/zephyr/build/zephyr/.config'
Kconfig header saved to '/home/user/zephyrproject/zephyr/build/zephyr/include/generated/zephyr/autoconf.h'
-- Found GnuLd: /usr/bin/ld.bfd (found version "2.38") 
-- The C compiler identification is GNU 11.4.0
-- The CXX compiler identification is GNU 11.4.0
-- The ASM compiler identification is GNU
-- Found assembler: /usr/bin/gcc
-- Found PkgConfig: /usr/bin/pkg-config (found version "0.29.2") 
-- Checking for one of the modules 'fuse'
-- Using ccache: /usr/local/bin/ccache
-- Found gen_kobject_list: /home/user/zephyrproject/zephyr/scripts/build/gen_kobject_list.py
-- Configuring done
-- Generating done
-- Build files have been written to: /home/user/zephyrproject/zephyr/build
-- west build: building application
[1/144] Preparing syscall dependency handling

[3/144] Generating include/generated/zephyr/version.h
-- Zephyr version: 4.1.99 (/home/user/zephyrproject/zephyr), build: v4.1.0-4740-g52f089af239b
[142/144] Linking C executable zephyr/zephyr.elf
Generating files from /home/user/zephyrproject/zephyr/build/zephyr/zephyr.elf for board: native_sim
[143/144] Building native simulator runner, and linking final executable
FAILED: zephyr/CMakeFiles/native_runner_executable zephyr/zephyr.exe /home/user/zephyrproject/zephyr/build/zephyr/CMakeFiles/native_runner_executable /home/user/zephyrproject/zephyr/build/zephyr/zephyr.exe 
cd /home/user/zephyrproject/zephyr/build/zephyr && /usr/bin/make -f /home/user/zephyrproject/zephyr/scripts/native_simulator/Makefile all --warn-undefined-variables -r NSI_CONFIG_FILE=/home/user/zephyrproject/zephyr/build/zephyr/NSI/nsi_config
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘ffa_run_pending_op’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:65:30: warning: implicit declaration of function ‘offsetof’ [-Wimplicit-function-declaration]
   65 | #define OP_RMDIR             offsetof(struct ffa_op_callbacks, rmdir)
      |                              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:113:14: note: in expansion of macro ‘OP_RMDIR’
  113 |         case OP_RMDIR:
      |              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:33:1: note: ‘offsetof’ is defined in header ‘<stddef.h>’; did you forget to ‘#include <stddef.h>’?
   32 | #include "fuse_fs_access_bottom.h"
  +++ |+#include <stddef.h>
   33 | 
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:65:39: error: expected expression before ‘struct’
   65 | #define OP_RMDIR             offsetof(struct ffa_op_callbacks, rmdir)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:113:14: note: in expansion of macro ‘OP_RMDIR’
  113 |         case OP_RMDIR:
      |              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:64:39: error: expected expression before ‘struct’
   64 | #define OP_UNLINK            offsetof(struct ffa_op_callbacks, unlink)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:116:14: note: in expansion of macro ‘OP_UNLINK’
  116 |         case OP_UNLINK:
      |              ^~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:63:39: error: expected expression before ‘struct’
   63 | #define OP_TRUNCATE          offsetof(struct ffa_op_callbacks, truncate)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:119:14: note: in expansion of macro ‘OP_TRUNCATE’
  119 |         case OP_TRUNCATE: {
      |              ^~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:62:39: error: expected expression before ‘struct’
   62 | #define OP_FTRUNCATE         offsetof(struct ffa_op_callbacks, ftruncate)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:125:14: note: in expansion of macro ‘OP_FTRUNCATE’
  125 |         case OP_FTRUNCATE: {
      |              ^~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:61:39: error: expected expression before ‘struct’
   61 | #define OP_WRITE             offsetof(struct ffa_op_callbacks, write)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:131:14: note: in expansion of macro ‘OP_WRITE’
  131 |         case OP_WRITE: {
      |              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:60:39: error: expected expression before ‘struct’
   60 | #define OP_READ              offsetof(struct ffa_op_callbacks, read)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:137:14: note: in expansion of macro ‘OP_READ’
  137 |         case OP_READ: {
      |              ^~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:59:39: error: expected expression before ‘struct’
   59 | #define OP_RELEASE           offsetof(struct ffa_op_callbacks, release)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:143:14: note: in expansion of macro ‘OP_RELEASE’
  143 |         case OP_RELEASE:
      |              ^~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:58:39: error: expected expression before ‘struct’
   58 | #define OP_CREATE            offsetof(struct ffa_op_callbacks, create)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:146:14: note: in expansion of macro ‘OP_CREATE’
  146 |         case OP_CREATE: {
      |              ^~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:57:39: error: expected expression before ‘struct’
   57 | #define OP_MKDIR             offsetof(struct ffa_op_callbacks, mkdir)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:152:14: note: in expansion of macro ‘OP_MKDIR’
  152 |         case OP_MKDIR:
      |              ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:56:39: error: expected expression before ‘struct’
   56 | #define OP_READDIR_END       offsetof(struct ffa_op_callbacks, readdir_end)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:155:14: note: in expansion of macro ‘OP_READDIR_END’
  155 |         case OP_READDIR_END:
      |              ^~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:55:39: error: expected expression before ‘struct’
   55 | #define OP_READDIR_READ_NEXT offsetof(struct ffa_op_callbacks, readdir_read_next)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:158:14: note: in expansion of macro ‘OP_READDIR_READ_NEXT’
  158 |         case OP_READDIR_READ_NEXT:
      |              ^~~~~~~~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:54:39: error: expected expression before ‘struct’
   54 | #define OP_READDIR_START     offsetof(struct ffa_op_callbacks, readdir_start)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:161:14: note: in expansion of macro ‘OP_READDIR_START’
  161 |         case OP_READDIR_START:
      |              ^~~~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:53:39: error: expected expression before ‘struct’
   53 | #define OP_READMOUNT         offsetof(struct ffa_op_callbacks, readmount)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:164:14: note: in expansion of macro ‘OP_READMOUNT’
  164 |         case OP_READMOUNT: {
      |              ^~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:52:39: error: expected expression before ‘struct’
   52 | #define OP_STAT              offsetof(struct ffa_op_callbacks, stat)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:170:14: note: in expansion of macro ‘OP_STAT’
  170 |         case OP_STAT: {
      |              ^~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_getattr’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:52:39: error: expected expression before ‘struct’
   52 | #define OP_STAT              offsetof(struct ffa_op_callbacks, stat)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:230:24: note: in expansion of macro ‘OP_STAT’
  230 |         err = queue_op(OP_STAT, (void *)&args);
      |                        ^~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_readmount’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:53:39: error: expected expression before ‘struct’
   53 | #define OP_READMOUNT         offsetof(struct ffa_op_callbacks, readmount)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:277:32: note: in expansion of macro ‘OP_READMOUNT’
  277 |                 err = queue_op(OP_READMOUNT, (void *)&args);
      |                                ^~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_readdir’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:54:39: error: expected expression before ‘struct’
   54 | #define OP_READDIR_START     offsetof(struct ffa_op_callbacks, readdir_start)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:324:32: note: in expansion of macro ‘OP_READDIR_START’
  324 |                 err = queue_op(OP_READDIR_START, (void *)mount_path);
      |                                ^~~~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:54:39: error: expected expression before ‘struct’
   54 | #define OP_READDIR_START     offsetof(struct ffa_op_callbacks, readdir_start)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:326:32: note: in expansion of macro ‘OP_READDIR_START’
  326 |                 err = queue_op(OP_READDIR_START, (void *)path);
      |                                ^~~~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:55:39: error: expected expression before ‘struct’
   55 | #define OP_READDIR_READ_NEXT offsetof(struct ffa_op_callbacks, readdir_read_next)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:351:32: note: in expansion of macro ‘OP_READDIR_READ_NEXT’
  351 |                 err = queue_op(OP_READDIR_READ_NEXT, (void *)&entry);
      |                                ^~~~~~~~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:56:39: error: expected expression before ‘struct’
   56 | #define OP_READDIR_END       offsetof(struct ffa_op_callbacks, readdir_end)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:373:18: note: in expansion of macro ‘OP_READDIR_END’
  373 |         queue_op(OP_READDIR_END, NULL);
      |                  ^~~~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_mkdir’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:57:39: error: expected expression before ‘struct’
   57 | #define OP_MKDIR             offsetof(struct ffa_op_callbacks, mkdir)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:382:28: note: in expansion of macro ‘OP_MKDIR’
  382 |         int err = queue_op(OP_MKDIR, (void *)path);
      |                            ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_create’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:58:39: error: expected expression before ‘struct’
   58 | #define OP_CREATE            offsetof(struct ffa_op_callbacks, create)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:401:24: note: in expansion of macro ‘OP_CREATE’
  401 |         err = queue_op(OP_CREATE, (void *)&args);
      |                        ^~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_release’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:59:39: error: expected expression before ‘struct’
   59 | #define OP_RELEASE           offsetof(struct ffa_op_callbacks, release)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:421:24: note: in expansion of macro ‘OP_RELEASE’
  421 |         (void)queue_op(OP_RELEASE, (void *)&fi->fh);
      |                        ^~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_read’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:60:39: error: expected expression before ‘struct’
   60 | #define OP_READ              offsetof(struct ffa_op_callbacks, read)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:443:24: note: in expansion of macro ‘OP_READ’
  443 |         err = queue_op(OP_READ, (void *)&args);
      |                        ^~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_write’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:61:39: error: expected expression before ‘struct’
   61 | #define OP_WRITE             offsetof(struct ffa_op_callbacks, write)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:465:24: note: in expansion of macro ‘OP_WRITE’
  465 |         err = queue_op(OP_WRITE, (void *)&args);
      |                        ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_ftruncate’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:62:39: error: expected expression before ‘struct’
   62 | #define OP_FTRUNCATE         offsetof(struct ffa_op_callbacks, ftruncate)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:484:24: note: in expansion of macro ‘OP_FTRUNCATE’
  484 |         err = queue_op(OP_FTRUNCATE, (void *)&args);
      |                        ^~~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_truncate’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:63:39: error: expected expression before ‘struct’
   63 | #define OP_TRUNCATE          offsetof(struct ffa_op_callbacks, truncate)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:497:24: note: in expansion of macro ‘OP_TRUNCATE’
  497 |         err = queue_op(OP_TRUNCATE, (void *)&args);
      |                        ^~~~~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_rmdir’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:65:39: error: expected expression before ‘struct’
   65 | #define OP_RMDIR             offsetof(struct ffa_op_callbacks, rmdir)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:504:28: note: in expansion of macro ‘OP_RMDIR’
  504 |         int err = queue_op(OP_RMDIR, (void *)path);
      |                            ^~~~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c: In function ‘fuse_fs_access_unlink’:
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:64:39: error: expected expression before ‘struct’
   64 | #define OP_UNLINK            offsetof(struct ffa_op_callbacks, unlink)
      |                                       ^~~~~~
/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.c:511:28: note: in expansion of macro ‘OP_UNLINK’
  511 |         int err = queue_op(OP_UNLINK, (void *)path);
      |                            ^~~~~~~~~
make: *** [/home/user/zephyrproject/zephyr/scripts/native_simulator/Makefile:113: /home/user/zephyrproject/zephyr/build/zephyr/NSI/home/user/zephyrproject/zephyr/subsys/fs/fuse_fs_access_bottom.o] Error 1
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/bin/cmake --build /home/user/zephyrproject/zephyr/build
```

</details>

**Environment (please complete the following information):**

 - OS: Linux Ubuntu 22.04
 - Toolchain: Zephyr sdk 0.17.1
 - Commit: 52f089af239bc9c5e1054de91a84429c0b60803b

**Additional context**
Adding `#include <stddef.h>` to `subsys/fs/fuse_fs_access_bottom.c` fixes the issue.

--- a/subsys/fs/fuse_fs_access_bottom.c
+++ b/subsys/fs/fuse_fs_access_bottom.c
@@ -10,6 +10,7 @@
 #undef _XOPEN_SOURCE
 #define _XOPEN_SOURCE 700
 
+#include <stddef.h>
 #include <stdbool.h>
 #include <stdio.h>
 #include <stdlib.h>

